packages/lesswrong/components/posts/PostsPage/PostsPageTwo.jsx

```diff
@@ -0,0 +1,306 @@
+    },
+    inline: {
+      display: 'inline-block'
+    },
+
+})
+
+class PostsPage extends Component {
+
+  render() {
+    const { loading, document, currentUser, location, router, classes, params } = this.props
+    const { PostsPageTitle, PostsAuthors, HeadTags, PostsVote, SmallMapPreviewWrapper,
+      LinkPostMessage, PostsCommentsThread, Loading, Error404, PostsGroupDetails, RecommendedReadingWrapper,
+      PostsTopSequencesNav, PostsPageMetadata, ModerationGuidelinesBox, HorizontalPostsVote, FromNowDate,
+      PostsPageMobileActions } = Components
+
+    if (loading) {
+      return <div><Loading/></div>
+    } else if (!document) {
+      return <Error404/>
+    } else {
+      const post = document
+      let query = location && location.query
+      const view = _.clone(router.location.query).view || Comments.getDefaultView(post, currentUser)
+      const description = post.plaintextExcerpt ? post.plaintextExcerpt : (post.body && post.body.substring(0, 300))
+      const commentTerms = _.isEmpty(query && query.view) ? {view: view, limit: 500} : {...query, limit:500}
+      const sequenceId = params.sequenceId || post.canonicalSequenceId;
+
+      return (
+        <div className={classes.root}>
+          <HeadTags url={Posts.getPageUrl(post, true)} title={post.title} description={description}/>
+
+          {/* Header/Title */}
+          <div className={classes.header}>
+            <PostsTopSequencesNav post={post} sequenceId={sequenceId} />
+            <PostsPageTitle post={post} />
+            <span className={classes.mobileVote}>
+              <Hidden mdUp implementation="css">
+                <PostsVote collection={Posts} post={post} currentUser={currentUser}/>
+              </Hidden>
+            </span>
+            <Hidden smDown implementation="css">
+              <div className={classes.voteTop}>
+                <PostsVote collection={Posts} post={post} currentUser={currentUser}/>
+              </div>
+            </Hidden>
+            {post.groupId && <PostsGroupDetails post={post} documentId={post.groupId} />}
+            <div className={classes.secondaryInfo}>
+              <span className={classes.inline}><PostsAuthors post={post}/></span>
+              <span className={classes.mobileDate}>
+                <Hidden mdUp implementation="css">
+                  <FromNowDate date={post.postedAt}/>
+                </Hidden>
+              </span>
+              <span className={classes.mobileActions}>
+                <Hidden mdUp implementation="css">
+                  <PostsPageMobileActions post={post} />
+                </Hidden>
+              </span>
+              <Hidden mdUp implementation="css">
+                <hr className={classes.mobileDivider} />
+              </Hidden>
+            </div>
+          </div>
+
+          {/* Body */}
+          <div className={classes.postBody}>
+            <Hidden smDown implementation="css">
+              <PostsPageMetadata post={post} />
```

Comment:
The breakpoints here have it so we use`PostsPageMetadata`if the screen size is <=sm, and a bunch of separate fields in the header if it's >sm. This seems error prone; since the fields are split between two files, it'd be pretty easy to accidentally omit a field in one or the other, and fail to notice the difference during a horizontal-resize test.

And if I'm reading it right (I haven't actually checked it out and looked at it yet), it looks like the event date/time fields are missing from the <=sm view.